### PR TITLE
[Node/VisualStudio] ignore typings directory

### PR DESCRIPTION
**Pull request by bbodenmiller**
**Reasons for making this change:**

Various tools now download these automatically

**Links to documentation supporting these rule changes:** 

https://github.com/Microsoft/nodejstools/wiki/Automatic-Typings-Acquisition
https://code.visualstudio.com/docs/runtimes/nodejs#_intellisense-and-typings
https://github.com/typings/typings#usage

### CODE REVIEW

**Comment by shiftkey** [comment]
@bbodenmiller with Typescript 2.0 moving away from `typings` to using `node_modules` for storing these files, this no longer seems important, right?

https://blogs.msdn.microsoft.com/typescript/2016/06/15/the-future-of-declaration-files/

**Comment by bbodenmiller** [comment]
I think it's going to take a bit of time before the `typings` directory is irreverent.

File: Node.gitignore [hunk 1]
```diff
@@ -30,6 +30,7 @@ build/Release
 # Dependency directories
 node_modules
 jspm_packages
+typings
```
Comment (shiftkey) [inline Node.gitignore]: Can you document this and indicate it's only relevant for Typescript pre-v2 projects?

**Comment by shiftkey** [comment]
@bbodenmiller I've just cleaned up things to get this in. Thanks!

**Comment by bbodenmiller** [comment]
👍

**Pull request merged**